tool/lint.sh

```diff
@@ -15,3 +15,17 @@ if [ -n "$bad" ]; then
 	echo "$bad"
 	exit 1
 fi
+
+bad=$(grep -Pzo '\)\s*\n\s*\{' src/main/c/cext/ruby.c || exit 0)
```

Comment:
Yeah, that one is a bit hacky, it uses Perl regexps and relies on GNU grep.
OTOH we only run lint.sh on Linux.
I could replace it with a snippet of Ruby code if you think that's better (basically normal grep doesn't allow a multi-lines pattern).